### PR TITLE
Editing while reading

### PR DESCRIPTION
**Pull request by jean**

### CODE REVIEW

**Comment by jamadden** [comment]
LGTM modulo the changelog note (the Travis/Mac build problems are an unrelated issue, AFAICS), but I'd appreciate a second set of eyeballs.

File: CHANGES.rst [hunk 1]
```diff
@@ -4,7 +4,7 @@ Changes
 4.3.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Editing while reading.
```
Comment (jamadden) [inline CHANGES.rst]: I don't find this very informative. Perhaps something more like "Correct typos and ReST formatting errors in documentation."

**Comment by icemac** [comment]
@jean LGTM. The MacOS failures seem to be problems on the side of Travis CI. Please update your PR with the current master so it can be merged.

**Comment by jean** [comment]
Done :-)

**Comment by tseaver** [comment]
@jean Thanks for the corrections!

**Pull request merged**